client/src/main/java/com/chattycathy/client/handler/ClientStompSessionHandler.java

```diff
@@ -27,12 +28,20 @@ public class ClientStompSessionHandler extends StompSessionHandlerAdapter {
     public void afterConnected(StompSession session, StompHeaders connectedHeaders) {
         log.info("Connected to WebSocket server");
         log.debug("Connected successfully to session {}, headers: {}", session, connectedHeaders);
+        Scanner s = new Scanner(System.in);
```

Comment:
```suggestion
        Scanner scanner = new Scanner(System.in);
```
(You will need to change all references not just accept this suggestion)
Just removes the need to look back up the code to find out what s is

Reply:
will update thanks